src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -5023,8 +5020,8 @@ namespace WindowEquivalentLayer {
                         Array1A<Real64> JF,           // returned: front (outside facing) radiosity of surfaces, W/m2
                         Array1A<Real64> JB,           // returned: back (inside facing) radiosity, W/m2
                         Array1A<Real64> HC,           // returned: gap convective heat transfer coefficient, W/m2K
-                        Real64 &UCG,                  // returned: center-glass U-factor, W/m2-K
-                        Real64 &SHGC,                 // returned: center-glass SHGC (Solar Heat Gain Coefficient)
+                        Optional<Real64> UCG,         // returned: center-glass U-factor, W/m2-K
+                        Optional<Real64> SHGC,        // returned: center-glass SHGC (Solar Heat Gain Coefficient)
```

Comment:
I'm pretty opposed to the use of optionals here.  Is there a reason they can't just be passed in dummy values?  Or if this function is used for two separate purposes, maybe you can find the common code and break it off into a separate one that's called from two different workers.  Then you only have to call the function with the arguments relevant to your current use case.

Reply:
Broke off ASHWAT_Thermal() function into two functions and eliminated the two optional arguments.